ab2.mjs

```
@@ -0,0 +1,21 @@
+process.env.NEUROLINK_SKIP_MCP="true";
+import { createServer } from "node:http";
+const { NeuroLink } = await import("./dist/index.js");
+for (const k of ["AWS_ACCESS_KEY_ID","AWS_SECRET_ACCESS_KEY","AWS_SESSION_TOKEN","SAGEMAKER_ENDPOINT"]) delete process.env[k];
```

ESLint

[error] Expected { after 'for-of'.

(curly)